src/Components/home/Calendar.jsx

```diff
@@ -1,64 +1,20 @@
 import React from "react";
-import API from "../../api";
-import FullCalendar from "@fullcalendar/react";
-import dayGridPlugin from "@fullcalendar/daygrid";
-import timeGridPlugin from "@fullcalendar/timegrid";
-import googleCalendarPlugin from "@fullcalendar/google-calendar";
-import interactionPlugin from "@fullcalendar/interaction";
-import tippy from "tippy.js";
-import "tippy.js/dist/tippy.css";
-import {darkOrange, darkBlue, darkGreen } from "../../theme";
-
-
-import "@fullcalendar/core/main.css";
-import "@fullcalendar/daygrid/main.css";
-import "@fullcalendar/timegrid/main.css";
-import "./calendar.css";
-
-const colors = [darkOrange, darkBlue, darkGreen];
-
-const calendarHeader = {
-  left: "prev,next today",
-  center: "title",
-  right: "dayGridMonth,timeGridWeek,timeGridDay,listWeek"
-};
-
-export function calendarCall(info, successCallback) {
-  API.get(`/calendar?start=${info.start}&end=${info.end}`)
-    .then((res) => successCallback(res.data))
-    .catch((err) => console.error(err));
-}
+import styled from "styled-components";
 
 export default function Calendar() {
-
   return (
-    <FullCalendar
-      defaultView="dayGridMonth"
-      eventTextColor="white"
-      eventBorderColor="rgba(0, 0, 0, 0)"
-      plugins={[dayGridPlugin, timeGridPlugin, googleCalendarPlugin, interactionPlugin]}
-      googleCalendarApiKey='AIzaSyBREtcqxmgY2SbJB5ddscMlKpWrHM0R1JM'
-      events={{googleCalendarId: "q2o6gp4ogd31rls2q7sqvl5nps@group.calendar.google.com"}}
-      header={calendarHeader}
-      eventDataTransform={(info) => {
-        info.url = undefined;
-      }}
-      eventRender={(info) => {
-        const {location, description} = info.event.extendedProps;
-        info.el.style.backgroundColor = colors[Math.floor(Math.random() * colors.length)];
-        info.el.setAttribute("data-tippy-content", `<strong>Location</strong>: ${location} <strong>Description</strong>: ${description}`); 
-        tippy(info.el, {
-          allowHTML: true,
-          trigger: "click",
-          arrow: true,
-          duration: 500,
-          interactive: true,
-          interactiveBorder: 30,
-          maxWidth: 500,
-          placement: "bottom",
-          appendTo: document.body
-        });
-      }}
-    />
+    <Container>
+      <iframe 
+        title="calendar" 
+        src="https://calendar.google.com/calendar/b/2/embed?height=600&amp;wkst=1&amp;bgcolor=%23ffffff&amp;ctz=America%2FNew_York&amp;showTitle=0&amp;showPrint=0&amp;src=Y2FsbGlicml0eS5jb21fcHE1dDVlYWNyczVoN29haHNhOWk3bTJtNm9AZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&amp;src=Y2FsbGlicml0eS5jb21fMGY5aXJjMDFrOGowZ3A5ZG9xbTRvZGpvcm9AZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&amp;src=Y2FsbGlicml0eS5jb21fYnM0b3U4NDB0dnVva2w2N2JhMG50MjZycGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&amp;color=%23F6BF26&amp;color=%23009688&amp;color=%23F6BF26" 
```

Comment:
What do you think about having the URL being a prop you can pass in so we can easily flip flop calendars?  I'd also prefer to have this url strings stored in a constants file or possibly a config allowing us to update it in a file that lives outside the build

Reply:
Or I could see us pulling those URL's from the database or something.  Possibly then building out a form allowing Jelena or whoever to update or add more calendars?  Thoughts?  Does that make sense?

Reply:
Can you host multiple calendars on the embedded widget? Would be nice to overlay the holiday calendar with the events calendar for example. Instead of needing some kind ability to change which one is loaded/shown.

Reply:
Looks like you can. From an account that is already able to view multiple calendars, you can choose which to include in the embedded src from the customization menu here:
![image](https://user-images.githubusercontent.com/64547264/83060867-1c2cf380-a02a-11ea-9afc-d472ea7fd9b1.png)

Reply:
As you saw, yes you can host multiple calendars at once by changing the source string.  Ideally we'll be able to build that string differently based on who logged in so they can view the calendars applicable to them.

Reply:
and I agree the src string should be separate.  Going to tinker more with that.  For some reason i was having trouble getting it to display when the string came from a variable.